components/airtable/actions/create-multiple-records/create-multiple-records.js

```diff
@@ -10,34 +10,49 @@ module.exports = {
   type: "action",
   props: {
     airtable,
-    baseId: {type: "$.airtable.baseId", appProp: 'airtable'},
-    tableId: { type: '$.airtable.tableId', baseIdProp: 'baseId' },
-    records: { propDefinition: [airtable, "records"] },
+    baseId: {
+      type: "$.airtable.baseId",
+      appProp: "airtable",
+    },
+    tableId: {
+      type: "$.airtable.tableId",
+      baseIdProp: "baseId",
+    },
+    records: {
+      propDefinition: [
+        airtable,
+        "records",
+      ],
+    },
   },
   methods: {
     async addRecords(records) {
-      const base = new Airtable({apiKey: this.airtable.$auth.api_key}).base(this.baseId);
-      return await base(this.tableId).create(records)
+      const base = new Airtable({
+        apiKey: this.airtable.$auth.api_key,
+      }).base(this.baseId);
+      return await base(this.tableId).create(records);
     },
   },
   async run() {
-    const records = []
-    let response_records = []
+    const records = [];
+    let responseRecords = [];
     const BATCH_SIZE = 10; // Airtable API allows to update up to 10 rows per request.
 
-    let inputRecords = this.records
+    let inputRecords = this.records;
 
-    if(!Array.isArray(inputRecords)) {
-      inputRecords = JSON.parse(inputRecords)
+    if (!Array.isArray(inputRecords)) {
+      inputRecords = JSON.parse(inputRecords);
     }
 
-    inputRecords.forEach(record => { records.push({ fields: record }) })
-    
-    const records_sets = chunk(records, BATCH_SIZE)
-    for (const records_set of records_sets) {        
-      response_records = response_records.concat((await this.addRecords(records_set)))
+    inputRecords.forEach(record => { records.push({
+      fields: record,
+    }); });
```

Comment:
I know it's out of scope but I'm going to leave a quick suggestion here to piggyback on this PR:

```suggestion
    const records = inputRecords.map((fields) => ({
      fields,
    }));
```